tests/ut/ir/transforms/test_flatten_call_expr_pass.py

```
@@ -601,11 +598,11 @@ def main(
                 out: pl.Out[pl.Tensor[[64], pl.FP32]],
             ) -> pl.Tensor[[64], pl.FP32]:
                 with pl.manual_scope():
-                    a = self.k1(x)
+                    a, a_tid = pl.submit(self.k1, x)
```

Ruff (0.15.12)

[warning] Unpacked variable `a` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)